app.js

```diff
@@ -0,0 +1,176 @@
+//  Main menu interactions
+//  Extract hamburguer menu icon object from DOM
+const menuIcon = document.getElementById('menuIcon');
+
+//  Create menu container
+const dynamicNav = document.createElement('div'); // Menu container
+dynamicNav.id = 'dynamicNav';
+
+//  Create closer icon
+const closerIcon = document.createElement('img');
+closerIcon.id = 'closerIcon';
+closerIcon.src = 'images/closerMenuIcon.png';
+closerIcon.alt = 'Closer Icon';
+
+// Create Links to the sections of the page
+const firstMenuElement = document.createElement('a');
+const secondMenuElement = document.createElement('a');
+const thirdMenuElement = document.createElement('a');
+const fourthMenuElement = document.createElement('a');
+const fifthMenuElement = document.createElement('a');
+
+firstMenuElement.innerText = 'About';
+secondMenuElement.innerText = 'Program';
+thirdMenuElement.innerText = 'Join';
+fourthMenuElement.innerText = 'Sponsors';
+fifthMenuElement.innerText = 'News';
+firstMenuElement.target = '-blank';
```

Comment:
- [OPTIONAL] After clicking on the `About` button to go to the **About** page, it redirects to a New Tab, but I think it will be better to stay on the same page after navigating from page to page what do you think?